frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/result-tab/pending-results.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/EmptyState/empty-state';
+import { EmptyState, EmptyStateBody, EmptyStateVariant, Title } from '@patternfly/react-core';
+import { LoadingInline } from '@console/internal/components/utils';
+import { iGetCommonData } from '../../selectors/immutable/selectors';
+import { VMWizardProps } from '../../types';
+import { makeSentence } from '../../../../utils/grammar';
+import { getVMLikeModelName } from '../../../../utils/utils';
+
+const PendingResultsComponent: React.FC<PendingResultsProps> = ({ isCreateTemplate }) => (
+  <EmptyState variant={EmptyStateVariant.full}>
+    <div className={css(styles.emptyStateIcon)}>
+      <LoadingInline />
+    </div>
+    <Title headingLevel="h5" size="lg">
+      {makeSentence(`${getVMLikeModelName(isCreateTemplate)} is being created.`)}
```

Comment:
The need for `makeSentence()` can be workarounded by i.e. by staring the sentence with `The`.

Reply:
There is no `The` in the design (http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/Create-vm/step-6-results/wizard-results) so I will leave it in the current state for now.

Reply:
resolved to a different solution